content/en/apps/guides/hosting/core-developer.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "CHT Core Developer Setup"
+linkTitle: "CHT Core Developer Setup"
+weight: 40
+description: >
+  Getting your local machine ready to to development work on CHT Core.
+---
+
+{{% alert title="Note" %}} This guide assumes you are a CHT Core developer wanting to run the CHT Core from source code to make commits to the [public GitHub repository](https://github.com/medic/cht-core). To set up a your environment for developing apps, see the [app guide]({{< relref "apps/guides/hosting/app-developer.md" >}}).
+
+To deploy the CHT in production, see either [AWS hosting]({{< relref "apps/guides/hosting/self-hosting.md" >}}) or [Self hosting]({{< relref "apps/guides/hosting/ec2-setup-guide.md" >}}){{% /alert %}}
+
+
+## The Happy Path Installation
+
+This CHT Core developer guide will have you install NodeJS, npm, Grunt and CouchDB (via Docker) on your local workstation. These instructions should work verbatim on Ubuntu 18-22, but will need tweaks for MacOS (via `brew`) or Windows (via WSL2).
+
+### Install NodeJS, npm, grunt and Docker
+
+First, update your current Ubuntu packages and install some supporting tools via `apt`:
+
+```shell
+sudo apt update && sudo apt -y dist-upgrade
+sudo apt -y install xsltproc curl uidmap jq
+```
+
+Then install `nvm`, add it to your path and install NodeJS 12:
+
+```shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | $0
```

Comment:
`0.34.0` is not the latest and the docs now recommend using `0.39.1`. But I wonder if we should just say `master` so we don't have to update this every time? Feels wrong though.

Reply:
Hmm - `curl` + `jq` = ftw? see 882adb2